lib/src/screens/crypto_currency_screen.dart

```diff
@@ -1,10 +1,113 @@
 import 'package:flutter/material.dart';
 
-class CryptoCurrencyScreen extends StatelessWidget {
+import '../config/app_constants.dart';
+import '../models/data.dart';
+import '../utils/custom_number_format.dart';
+import '../widgets/heading_row.dart';
+import '../constants/constants.dart' as constants;
+
+class CryptoCurrencyScreen extends StatefulWidget {
   const CryptoCurrencyScreen({Key? key}) : super(key: key);
 
+  @override
+  State<CryptoCurrencyScreen> createState() => _CryptoCurrencyScreenState();
+}
+
+class _CryptoCurrencyScreenState extends State<CryptoCurrencyScreen>
+    with AutomaticKeepAliveClientMixin {
+
+  double columnSpacing = 18;
+  double horizontalMargin = 2;
+  double dataRowHeight = 50;
+  double headingRowHeight = 50;
+  double widthOfImage = 20;
+  double heightOfImage = 20;
```

Comment:
they can't be accessed from anywhere else so make it private columnSpacing => _columnSpacing

Reply:
i fixed it private